clients/restclient/rest_client.go

```diff
@@ -79,6 +81,12 @@ func getRestURL(org, space string) string {
 	return "rest/" + org + "/" + space
 }
 
+func getRestURLEncoded(org, space string) string {
+	orgEncoded := strings.Replace(url.QueryEscape(org), "+", "%20", -1)
```

Comment:
Could this and the below row be extracted in a common method, so that they can be re-used?

Reply:
The real encoding is extracted in new function client_util->EncodeArg()